pkg/worker/events.go

```diff
@@ -119,7 +120,7 @@ func (m *ProcessMonitor) fetchGPUMemory() *GPUInfoStat {
 		if device.Minor == nil || !device.Allow {
 			continue
 		}
-		stats, err := GetGpuMemoryUsage(int(*device.Minor))
+		stats, err := m.gpuInfoClient.GetGPUMemoryUsage(int(*device.Minor))
```

Comment:
Were you able to confirm using Minor works for multi-gpu nodes? If so, this is great.

Reply:
I updated the function to use the device obtained from `AvailableDevices`. This way it's consistent with the code that assigned it.
```
func (m *ProcessMonitor) fetchGPUMemory() *GPUInfoStat {
	stat := &GPUInfoStat{}
	availableDevices, err := m.gpuInfoClient.AvailableGPUDevices()
	if err != nil {
		return stat
	}

	for _, device := range availableDevices {
		stats, err := m.gpuInfoClient.GetGPUMemoryUsage(device)
		if err == nil {
			stat.MemoryUsed += uint64(stats.UsedCapacity)
			stat.MemoryTotal += uint64(stats.TotalCapacity)
		}
	}
	return stat
}
```